src/main/java/org/icgc/argo/workflow_management/streams/DisposableManager.java

```diff
@@ -16,24 +16,26 @@
  * ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-package org.icgc.argo.workflow_management.properties;
+package org.icgc.argo.workflow_management.streams;
```

Comment:
not sure why git couldn't figure this out. This file already existed just moved rabbit->streams